lms/djangoapps/verify_student/tests/fake_software_secure.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+Fake Software Secure page for use in acceptance tests.
+"""
+from django.core.urlresolvers import reverse
+from django.views.generic.base import View
+from django.views.decorators.csrf import csrf_exempt
+from django.conf import settings
+
+from email.utils import formatdate
+from edxmako.shortcuts import render_to_response
+
+
+
+class SoftWareSecureFakeView(View):
+    """
+    Fake Softwaresecure page for use to change the verification status and check the emails functionality
```

Comment:
Please try to keep maximum length of docstring to 80, if length increases then break it down to multiple lines.
Change docstring to:
`Fake SoftwareSecure view for testing different photo verification statuses and email functionality.`